### PR TITLE
Fix version info and update from 1.1.8 to 1.1.9 for CIRCexplorer

### PR DESCRIPTION
**Pull request by kepbod**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

* Fix version info using real version of CIRCexplorer
* Upgrade from CIRCexplorer 1.1.8 to 1.1.9

### CODE REVIEW

**Comment by bgruening** [comment]
Looks great!

File: recipes/circexplorer/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 package:
   name: circexplorer
-  version: "0.1"
+  version: "1.1.9"
 
 build:
   number: 1
```
Comment (bgruening) [inline recipes/circexplorer/meta.yaml]: Can you please decrease the build number to 0.
Thanks1

Comment (kepbod) [inline recipes/circexplorer/meta.yaml]: OK. I have decreased it.

**Comment by bgruening** [comment]
Thanks a lot! Let me know if you want to have commit access: https://github.com/bioconda/bioconda-recipes/issues/1

**Comment by kepbod** [comment]
Of course that I would like to participate. Thanks!

**Comment by bgruening** [comment]
Welcome @kepbod!

**Pull request merged**